pennylane/devices/default_gaussian.py

```diff
@@ -572,7 +572,8 @@ def poly_quad_expectations(cov, mu, wires, device_wires, params, hbar=2.0):
 
     # HACK, we need access to the Poly instance in order to expand the matrix!
     # TODO: maybe we should make heisenberg_obs a class method or a static method to avoid this being a 'hack'?
-    op = qml.ops.PolyXP(Q, wires=wires)
+    # static analysis can misidentify qml.ops as the set instance qml.ops.qubit.ops
```

Comment:
This suggestion if accepted applies to other comments in this PR too.
```suggestion
    # Linting check disabled as static analysis can misidentify qml.ops as the set instance qml.ops.qubit.ops
```

Reply:
Sounds good, I'll add this here and in the other places.

Ideally it would nice to fix the cause of this issue (maybe something about our imports?) but I couldn't identify why this is happening.